src/test/java/edu/cornell/kfs/pdp/batch/service/impl/PayeeACHAccountExtractServiceImplTest.java

```diff
@@ -724,5 +724,60 @@ public void markRowAsInvalid() {
             this.validRow = false;
         }
     }
+    
+    @Test
+    public void testCleanPayeeACHAccountExtractDetailNoClean() {
+        String bankAccountNumber = "12345";
+        PayeeACHAccountExtractDetail detail = new PayeeACHAccountExtractDetail();
+        detail.setBankAccountNumber(bankAccountNumber);
+        payeeACHAccountExtractService.cleanPayeeACHAccountExtractDetail(detail);
+        
+        String expectedBankAccount = bankAccountNumber;
+        assertEquals(expectedBankAccount, detail.getBankAccountNumber());

```

Comment:
These new unit test methods have a lot of duplicated code. I would suggest creating a separate helper method to do the bulk of the setup and assertion work.

Reply:
good point, I shall refactor